applications/nrf_desktop/configuration/nrf52840_pca10059/app_ZRelease.conf

```diff
@@ -70,6 +70,11 @@ CONFIG_SYS_PM_POLICY_APP=y
 
 CONFIG_GPIO=y
 
+CONFIG_SERIAL=n
+CONFIG_UART_0_NRF_UARTE=n
+CONFIG_UART_INTERRUPT_DRIVEN=n
+CONFIG_CONSOLE=n
```

Comment:
Do we want to disable `USE_SEGGER_RTT` here?

Reply:
It is not enabled by board config so we don't have to